docs/decision-maker-transaction.md

```diff
@@ -14,7 +14,7 @@ from aea.crypto.cosmos import CosmosCrypto
 from aea.crypto.helpers import create_private_key
 from aea.crypto.ledger_apis import LedgerApis
 from aea.crypto.wallet import Wallet
-from aea.helpers.dialogue.base import Dialogue
+from aea.protocols.dialogue.base import Dialogue
```

Comment:
wrong position; needs to be moved down,


same below

Reply:
Done